test/update-spec-tests.py

```diff
@@ -92,6 +92,7 @@ def main(args):
     ProcessProposalDir('sign-extension-ops', '--enable-sign-extension')
     ProcessProposalDir('bulk-memory-operations', '--enable-bulk-memory')
     ProcessProposalDir('reference-types', '--enable-reference-types')
+    ProcessProposalDir('simd', '--enable-simd')
 
```

Comment:
Oops.. is this why there were so many new tests added here?  I guess we were not updating the simd tests previously?

Reply:
Yep, I guess there weren't SIMD tests for a while...